### PR TITLE
misc: use micron and dbu converters from odb

### PR DESCRIPTION
**Pull request by eder-matheus**

### CODE REVIEW

File: src/grt/src/GlobalRouter.cpp [hunk 1]
```diff
@@ -3907,7 +3907,7 @@ void GlobalRouter::reportLayerWireLengths()
         odb::dbTechLayer* layer = routing_layers_[i];
         logger_->report("{:5s} {:8d}um {:3d}%",
                         layer->getName(),
-                        static_cast<int64_t>(dbuToMicrons(length)),
+                        static_cast<int64_t>(block_->dbuToMicrons(length)),
```
Comment (rovinski) [inline src/grt/src/GlobalRouter.cpp:3910]:
```suggestion
                        block_->dbuToMicrons(length),
```

**Review by rovinski — COMMENTED** [review]
It looks to me like the antenna checker needs a wholesale conversion to DBU, because I don't see why it's using doubles right now.

**Comment by maliberty** [comment]
> It looks to me like the antenna checker needs a wholesale conversion to DBU, because I don't see why it's using doubles right now.

@luis201420 is in the process of rewriting it so hopefully this will be better.

**Review by maliberty — APPROVED** [review]
I agree with @rovinski 's comment but otherwise fine.

**Pull request merged**